### DISCUSSION TITLE
dear peppy

### DISCUSSION THREAD

**Discussion by leofancyngrok** [ppy/osu#17463] [Q&A] [answered]
so i have started playing on the Lazer version 
but want to use the (in-game driver) but cant figure out how to put in the same area if you can help me with that i would be very happy
![Screenshot 2022-03-26 170826](https://user-images.githubusercontent.com/38032102/160247886-15e60002-f7f9-4a46-9fc7-0c82cd755c93.png)

**Comment by HelloYeew** [comment, marked as answer]
Here is a rough guide. Just hold the mouse on the slider and check that number on tooltip is match. (Aspect ratio is calculate from width and height so just make sure that your width and height is match your setting)
![Screenshot 2022-03-26 232704](https://user-images.githubusercontent.com/68165621/160248534-4e5520e6-57da-4a0f-8a73-601431a9b9e9.png)
